src/mixxx.cpp

```diff
@@ -124,6 +125,14 @@ MixxxMainWindow::MixxxMainWindow(QApplication* pApp, const CmdlineArgs& args)
     // after an upgrade and make any needed changes.
     m_pUpgrader = new Upgrade;
     m_pConfig = m_pUpgrader->versionUpgrade(args.getSettingsPath());
+
+    registerConfigPath(m_pConfig->getSettingsPath() + "/Mixxx.cfg");
+    MixxxSettings mixxx_settings;
+    // I need to explicitely ask for a reference because QSettings doesn't
+    // support move in Qt4
+    auto& settings = mixxx_settings.getQSettings();
+    settings.setFallbacksEnabled(false);
```

Comment:
setFallbacksEnabled is non const, we should use the pointer notation.
A reference should be always const.